src/lib/stores/parent-resource.store.ts

```diff
@@ -11,3 +12,16 @@ export const parentResourceNameToInfoMap = derived(parentResources, ($parentReso
         (r) => r[0]
     )
 );
+
+export const guideResources = derived(parentResources, ($parentResources) =>
+    $parentResources.filter((r) => r.resourceType === 'Guide')
+);
+
+export const currentGuide = writable<ApiParentResource | undefined>(undefined);
```

Comment:
Would we want this to be pre-populated by the localStorage value? Might be nice to avoid calling into localStorage from components/pages and keep all that logic here. Example: https://github.com/BiblioNexusStudio/content-manager-web/blob/master/src/lib/stores/resources.ts#L6